Build.pm

```diff
@@ -159,13 +159,13 @@ my %subst_defaults = (
     'apt-utils', 'cpio', 'dpkg-dev', 'live-build', 'lsb-release', 'tar',
   ],
   'system-packages:mkosi:rpm' => [
-    'mkosi',
+    'mkosi', 'createrepo', 'gzip',
   ],
   'system-packages:mkosi:deb' => [
-    'mkosi',
+    'mkosi', 'dpkg-dev', 'lsb-release', 'gzip',
   ],
   'system-packages:mkosi:arch' => [
-    'mkosi',
+    'mkosi', 'arch-install-scripts', 'gzip',
```

Comment:
`arch-installs-scripts` can be dropped, we don't use it anymore in mkosi and haven't for a very long time.

Reply:
This is not about what mkosi uses, but what the glue script in this repo uses. mkosi can pull in its own dependencies, but this needs to pull in what the script uses itself, so that it's guaranteed to be there